src/rastervision/ml_backends/tf_deeplab.py

```diff
@@ -0,0 +1,436 @@
+import io
+import os
+import glob
+import numpy as np
+import shutil
+import tarfile
+import tempfile
+import tensorflow as tf
+import uuid
+
+from os.path import join
+from PIL import Image
+from subprocess import Popen
+from tensorflow.core.example.example_pb2 import Example
+from typing import (List, Tuple)
+from urllib.parse import urlparse
+
+from object_detection.utils import dataset_util
+from rastervision.core.box import Box
+from rastervision.core.class_map import ClassMap
+from rastervision.core.ml_backend import MLBackend
+from rastervision.core.scene import Scene
+from rastervision.core.training_data import TrainingData
+from rastervision.ml_backends.tf_object_detection_api import (
+    write_tf_record, terminate_at_exit, TRAIN, VALIDATION)
+from rastervision.utils.misc import save_img
+from rastervision.utils.files import (get_local_path, upload_if_needed,
+                                      make_dir, download_if_needed, sync_dir,
+                                      start_sync)
+
+
+def numpy_to_png(array: np.ndarray) -> str:
```

Comment:
Perhaps these are broadly useful enough to pull into a utility package?

Reply:
Okay, will-do.

Reply:
Done.